operator/Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM mcr.microsoft.com/oss/go/microsoft/golang:1.22 AS builder
 
 ARG VERSION
 ARG APP_INSIGHTS_ID
+RUN apt-get update && apt-get install -y ca-certificates
+RUN update-ca-certificates
```

Comment:
Mind combining these into one `RUN` line and pruning the byproducts that are not the ca-certificates?